core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.html

```diff
@@ -8,13 +8,9 @@
 </div>
 <div *ngIf="!languageSelected" class="oppia-language-required-message-container">
   <div class="oppia-mascot-avatar-container">
-    <a [ngClass]="{'oppia-disabled-image-link': OPPIA_AVATAR_LINK_URL === null}"
-       [href]="OPPIA_AVATAR_LINK_URL"
-       rel="noopener"
-       target="_blank">
-      <img class="oppia-mascot-avatar"
-           [src]="OPPIA_AVATAR_IMAGE_URL" alt="">
-    </a>
+    <img class="oppia-mascot-avatar"
```

Comment:
Go to the Translate Text tab and show what you see without selecting a language from the language selector,

Reply:
@chris7716 this I got,

![mentu](https://user-images.githubusercontent.com/118007662/216042586-5bd5472f-b05a-45ba-8edd-d158a57e730f.png)